src/synapse/Player.php

```diff
@@ -194,7 +194,9 @@ protected function processLogin(){
 			$pk->y = $this->y;
 			$pk->z = $this->z;
 			$pk->seed = -1;
-			$pk->dimension = $this->level->getDimension();
+			if($this->server->getName() != "PocketMine-MP"){
```

Comment:
and what happens when it's a spoon of PocketMine-MP which doesn't have these methods?